SessionGeneratorService/src/main/java/com/garymace/session/generator/service/SessionGeneratorApplication.java

```diff
@@ -30,5 +38,17 @@ public void run(
     SessionGeneratorServiceConfiguration sessionGeneratorServiceConfiguration,
     Environment environment
   )
-    throws Exception {}
+    throws Exception {
+    // Enable CORS headers
+    FilterRegistration.Dynamic cors =
+        environment.servlets().addFilter("CORS", CrossOriginFilter.class);
+
+    // Configure CORS parameters
+    cors.setInitParameter("allowedOrigins", "*");
+    cors.setInitParameter("allowedHeaders", "X-Requested-With,Content-Type,Accept,Origin");
+    cors.setInitParameter("allowedMethods", "OPTIONS,GET,PUT,POST,DELETE,HEAD");
+
+    // Add URL mapping
+    cors.addMappingForUrlPatterns(EnumSet.allOf(DispatcherType.class), true, "/*");
+  }
```

Comment:
Requests from the default 3000 react localhost port to the default dropwizard 8080 localhost port will get rejected by CORS otherwise.

I don't care enough to change this right now to be stricter but adding this note to review this later.